wsmaster/che-core-api-agent/src/main/resources/agents/org.eclipse.che.ls.csharp.json

```diff
@@ -3,5 +3,5 @@
ll 2>&1 || {\n curl -L -o dotnet.tar.gz https://go.microsoft.com/fwlink/?LinkID=809130;\n ${SUDO} apt-get update;\n ${SUDO} apt-get -y install libunwind8 gettext;\n ${SUDO} mkdir -p /opt/dotnet;\n ${SUDO} tar zxf dotnet.tar.gz -C /opt/dotnet;\n rm dotnet.tar.gz;\n ${SUDO} ln -s /opt/dotnet/dotnet /usr/local/bin;\n }\n\n command -v nodejs >/dev/null 2>&1 || {\n {\n if test \"${SUDO}\" = \"\"; then\n curl -sL https://deb.nodesource.com/setup_6.x | bash -;\n else\n curl -sL https://deb.nodesource.com/setup_6.x | ${SUDO} -E bash -;\n fi\n };\n\n ${SUDO} apt-get update;\n ${SUDO} apt-get install -y nodejs;\n }\n\n# Fedora 23\n###########\nelif echo ${LINUX_TYPE} | grep -qi \"fedora\"; then\n PACKAGES=${PACKAGES}\" procps-ng\"\n test \"${PACKAGES}\" = \"\" || {\n ${SUDO} dnf -y install ${PACKAGES};\n }\n\n command -v dotnet >/dev/null 2>&1 || {\n curl -L -o dotnet.tar.gz https://go.microsoft.com/fwlink/?LinkID=816869;\n ${SUDO} dnf -y install libunwind libicu;\n ${SUDO} mkdir -p /opt/dotnet;\n ${SUDO} tar zxf dotnet.tar.gz -C /opt/dotnet;\n rm dotnet.tar.gz;\n ${SUDO} ln -s /opt/dotnet/dotnet /usr/local/bin;\n }\n\n command -v nodejs >/dev/null 2>&1 || {\n curl --silent --location https://rpm.nodesource.com/setup_6.x | ${SUDO} bash -;\n ${SUDO} dnf -y install nodejs;\n }\n\n\n\n# CentOS 7.1 & Oracle Linux 7.1\n###############################\nelif echo ${LINUX_TYPE} | grep -qi \"centos\"; then\n test \"${PACKAGES}\" = \"\" || {\n ${SUDO} yum -y install ${PACKAGES};\n }\n\n command -v dotnet >/dev/null 2>&1 || {\n curl -L -o dotnet.tar.gz https://go.microsoft.com/fwlink/?LinkID=809131;\n ${SUDO} yum -y install libunwind libicu;\n ${SUDO} mkdir -p /opt/dotnet;\n ${SUDO} tar zxf dotnet.tar.gz -C /opt/dotnet;\n rm dotnet.tar.gz;\n ${SUDO} ln -s /opt/dotnet/dotnet /usr/local/bin;\n }\n\n command -v nodejs >/dev/null 2>&1 || {\n curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -;\n ${SUDO} yum -y install nodejs;\n }\n\n\n# openSUSE 13.2\n###############\nelif echo ${LINUX_TYPE} | grep -qi \"opensuse\"; then\n test \"${PACKAGES}\" = \"\" || {\n ${SUDO} zypper install -y ${PACKAGES};\n }\n\n command -v dotnet >/dev/null 2>&1 || {\n curl -L -o dotnet.tar.gz https://go.microsoft.com/fwlink/?LinkID=816867;\n ${SUDO} zypper install -y libunwind libicu;\n ${SUDO} mkdir -p /opt/dotnet;\n ${SUDO} tar zxf dotnet.tar.gz -C /opt/dotnet;\n rm dotnet.tar.gz;\n ${SUDO} ln -s /opt/dotnet/dotnet /usr/local/bin;\n }\n\n command -v nodejs >/dev/null 2>&1 || {\n ${SUDO} zypper ar http://download.opensuse.org/repositories/devel:/languages:/nodejs/openSUSE_13.1/ Node.js\n ${SUDO} zypper in nodejs\n }\n\nelse\n >&2 echo \"Unrecognized Linux Type\"\n >&2 cat /etc/os-release\n exit 1\nfi\n\n\n#####################\n### Install C# LS ###\n#####################\n\ncurl -s ${AGENT_BINARIES_URI} | tar xzf - -C ${CHE_DIR}\n\ntouch ${LS_LAUNCHER}\nchmod +x ${LS_LAUNCHER}\necho \"nodejs ${LS_DIR}/node_modules/omnisharp-client/languageserver/server.js\" > ${LS_LAUNCHER}"
```

Comment:
yes maybe add another issue for "pure Alpine" because users will start to use "official docker images" or "based images" that are switching to Alpine

for example openjdk:8-alpine has not bash inside